consensus/src/consensus.rs

```diff
@@ -182,15 +239,14 @@ impl<R: ConsensusRpc> ConsensusClient<R> {
         for update in updates {
             self.verify_update(&update)?;
             self.apply_update(&update);
-            self.store
-                .all_finalized_updates
-                .insert(update.finalized_header.slot, update.clone());
+            self.store_finalized_update(&update, false)?;
```

Comment:
> dose `storage.put_finalized_update(slot, &update)?;` in here support discontinuous slots? because the `update` in `updates` here does not keep continuous with the previous and the next one.

- The API here is `fn store_finalized_update(&self, update: &Update, update_mmr: bool) -> Result<()>;`.

- The continuous of updates is only checked when `update_mmr == true`.

Reply:
so the `storage.put_finalized_update(slot, &update)?` here won't be able to check the continuation of the key here named `slot`?

Reply:
> so the `storage.put_finalized_update(slot, &update)?` here won't be able to check the continuation of the key here named `slot`?

Yep!